src/main/java/org/apache/skywalking/banyandb/v1/client/BanyanDBClient.java

```diff
@@ -136,8 +155,10 @@ public void connect() {
                 nettyChannelBuilder.maxInboundMessageSize(options.getMaxInboundMessageSize());
 
                 channel = nettyChannelBuilder.build();
+                measureServiceStub = MeasureServiceGrpc.newStub(channel);
                 streamServiceStub = StreamServiceGrpc.newStub(channel);
-                streamServiceBlockingStub = StreamServiceGrpc.newBlockingStub(channel);
+                streamServiceBlockingStub = StreamServiceGrpc.newFutureStub(channel);
```

Comment:
```suggestion
                streamServiceBlockingStub = StreamServiceGrpc.newBlockingStub(channel);
```

According to the codes, this should be `blockingStub`?

Reply:
It should be `futureStub` not `blockingStub`. Names of fields have been changed.

Reply:
Why don't use blockingStub? It is much easier for no streaming service? Could you recheck?

Reply:
```java
Commands commands = stub.withDeadlineAfter(GRPC_UPSTREAM_TIMEOUT, TimeUnit.SECONDS)
                                            .collect(builder.build());
```

For a blocking stub, one line is enough.

Reply:
reverted to blocking apis